vendor/wipro/default-en.json

```diff
@@ -0,0 +1,14 @@
+{
+  "VENDOR" : "true",
+  "SIDEBAR": {
+    "HOME": "Overview",
+    "DASHBOARD": "Ci/E2E Dashboard",
+    "WORKLOADS": "Workload Dashboard",
+    "ORGANISATION": "MayaData.Inc",
```

Comment:
It should be wipro

Reply:
That will modify to wipro more thing has to be modify that will come in next PR!!